EpiOAuth.php

```diff
@@ -384,6 +390,8 @@ public function __construct($consumerKey, $consumerSecret, $signatureMethod='HMA
     $this->consumerKey = $consumerKey;
     $this->consumerSecret = $consumerSecret;
     $this->signatureMethod = $signatureMethod;
+    $this->serverAddr = (!empty($_SERVER['SERVER_ADDR']) && $_SERVER['SERVER_ADDR'] != '127.0.0.1')
```

Comment:
Any reason to set this conditionally if there's a method to set it explicitly? Is this to try and default it to the **correct** value?